tutorials/Creating a new Lumigator endpoint.md

```diff
@@ -0,0 +1,581 @@
+# Creating a new Lumigator endpoint
+
+This document shows how current endpoints in lumigator work and describes the process needed to create a new one. This
+might or might not become part of the public documentation - I think it might be useful for users to expand the tool
+with their own use cases and would allow us to grow more easily, but I am happy to collect feedback about it here first.
+
+## Lumigator's architecture
+
+The diagram below shows the current architecture of Lumigator. The larger
+containers (UI, Backend, Ray cluster, DB, Storage) are the different services Lumigator relies on.
```

Comment:
suggestion: link to the larger explanation here or copy it https://github.com/mozilla-ai/lumigator/tree/main/.devcontainer#running-lumigator-locally-with-docker-compose